apps/connection/src/connection.controller.ts

```diff
@@ -38,7 +38,7 @@ export class ConnectionController {
    * @returns Callback URL for connection and created connections details
    */
   @MessagePattern({ cmd: 'webhook-get-connection' })
-  async getConnectionWebhook(payload: IConnectionInterface): Promise<object> {
+  async getConnectionWebhook(payload: Connection): Promise<object> {
```

Comment:
Replace object as a return type with appropriate interface.

Reply:
It's not clear what data will come from the webhook, so its return type has been kept as an object. This was decided after a discussion with Ankita Ma'am and you, that for the webhook's APIs, the return type should remain as an object.